warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -7891,7 +7891,13 @@ public static String buildSelector(GeoTimeSerie gts) {
   
   public static String buildSelector(Metadata metadata) {
     StringBuilder sb = new StringBuilder();
+
+    char nameFirstChar = metadata.getName().charAt(0);
```

Comment:
This could fail with a GTS with an empty class name

Reply:
My bad.